### DISCUSSION TITLE
Where to fill in the password

### DISCUSSION THREAD

**Discussion by felix0080** [Grokzen/redis-py-cluster#468] [Q&A] [answered]
Where to fill in the password,I can't find a place to pass the password param

**Comment by Grokzen** [comment, marked as answer]
@felix0080 This lib supports all args and kwargs that redis-py supports in their class `Redis` and on top of that you have the following example showing how it works here https://github.com/Grokzen/redis-py-cluster/blob/master/examples/basic_password_protected.py

**Reply by felix0080** [reply]
That helped me, thank you
